scripts/bootstrap.sh

```diff
@@ -36,6 +36,15 @@ make
 popd
 cp minidump-stackwalk/stackwalker stackwalk/bin
 
+# setup any unset test configs and databases without overwriting existing files
+pushd config
+for file in *.ini-dist; do
+    if [ ! -f `basename $file -dist` ]; then
+        cp $file `basename $file -dist`
+    fi
+done
+popd
+
```

Comment:
If we no longer are going to depend on a `config/dataservice.ini` file, can we move this back into `scripts/test.sh`